Bundle/BlogBundle/Entity/Category.php

```diff
@@ -35,6 +35,7 @@ class Category
      *
      * @ORM\Column(name="slug", type="string", length=255)
      * @Gedmo\Slug(fields={"title"}, updatable=true, unique=false)
+     * @Vic\BusinessProperty("businessParameter")
```

Comment:
@naomiHauret Please add this line after the other `use`:

    use Victoire\Bundle\CoreBundle\Annotations as Vic;

Then tests should pass.